src/fsharp/FSharp.Compiler.Service/FSharp.Compiler.Service.netcore.fsproj

```diff
@@ -0,0 +1,615 @@
+<Project Sdk="FSharp.NET.Sdk;
+Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <VersionPrefix>11.0.8</VersionPrefix>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <DefineConstants>
+      $(DefineConstants);
```

Comment:
Please use the one-lin=-per-define style 
```
    <DefineConstants>$(DefineConstants);COMPILER_SERVICE</DefineConstants>
```
that's used in other F# project files